scenarios/media_and_entertainment/README.md

```diff
@@ -0,0 +1,39 @@
+<!--
+Copyright (c) Recommenders contributors.
+Licensed under the MIT License.
+-->
+
+# Recommendation systems for Media and Entertainment
+
+Recommendation systems are fundamental to modern media and entertainment platforms. For example, Netflix reports that their recommendation system [saves them $1 billion per year](https://www.netflix.com/tudum/articles/netflix-recommendations) in customer retention.
```

Comment:
This link is broken.

Reply:
@anargyri PR fixing it and adding more context: https://github.com/recommenders-team/recommenders/pull/2208